packages/cli-lib/lib/spawnFunction.js

```diff
@@ -3,7 +3,6 @@ const cp = require('child_process');
 const spawn = codeToExec => {
   // TODO: Sanitize input here, probably.
   const child = cp.spawnSync('node', ['-e', codeToExec], {
-    detached: true, //child process does not end when parent ends
```

Comment:
Behavior with this option differs on Windows. Instead of a different process it creates a new shell.

According to https://nodejs.org/api/child_process.html#optionsdetached: 
>  If the parent's stdio is inherited, the child will remain attached to the controlling terminal.
So I believe it was being ignored anyways